employees/common/exports.py

```diff
@@ -199,7 +206,9 @@ def _summarize_user_reports(self):
         total_hours_cell.number_format = constants.TOTAL_HOURS_FORMAT.value
         set_format_styles_for_main_cells(total_hours_cell, is_header=False)
 
-    def _get_report_date_and_daily_hours(self, current_report: Report, reports_subset: ReportQuerySet):
+    def _get_report_date_and_daily_hours(
+        self, current_report: Report, reports_subset: ReportQuerySet
+    ) -> Tuple[Optional[datetime], Optional[Decimal]]:
```

Comment:
I can see we are still using Decimal to represent work hours sum. Should this be changed as well now that we're using the time delta to represent hours?

Reply:
Yes, I saw it too, but for this PR I did not change it. If you can please add issue for that @maciejSamerdak